selfdrive/ui/qt/offroad/replay_controls.cc

```diff
@@ -0,0 +1,256 @@
+#include "selfdrive/ui/qt/offroad/replay_controls.h"
+
+#include <QDir>
+#include <QFuture>
+#include <QFutureWatcher>
+#include <QtConcurrent>
+#include <algorithm>
+#include <utility>
+
+#include "selfdrive/ui/ui.h"
+
+namespace {
+// TODO: move to assert file
+QString pause_svg = R"(<svg width="512" height="512" viewBox="0 0 512 512" style="color:#ffffff" xmlns="http://www.w3.org/2000/svg" class="h-full w-full"><rect width="512" height="512" x="0" y="0" rx="30" fill="transparent" stroke="transparent" stroke-width="0" stroke-opacity="100%" paint-order="stroke"></rect><svg width="125px" height="125px" viewBox="0 0 14 14" fill="#ffffff" x="193.5" y="193.5" role="img" style="display:inline-block;vertical-align:middle" xmlns="http://www.w3.org/2000/svg"><g fill="#ffffff"><g fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round"><circle cx="7" cy="7" r="6.5"/><path d="M5.5 4.5v5m3-5v5"/></g></g></svg></svg>)";
```

Comment:
Can't upload to lfs storage due to permission issues. So for now, save svg in strings.